mobile/src/main/java/ch/epfl/sdp/mobile/infrastructure/persistence/store/Query.kt

```diff
@@ -40,6 +40,17 @@ interface Query {
    */
   fun whereNotEquals(field: String, value: Any?): Query
 
+  /**
+   * Filters the results by keeping only the documents which contain a string value prefixed with
+   * the given [prefix] for the given [field].
+   *
+   * @param field the field for which the difference is checked.
+   * @param prefix the prefix that the value should start with.
+   *
+   * @return the updated [Query].
+   */
+  fun startsWith(field: String, prefix: String): Query
```

Comment:
You don't need this method to implement user following. Additionally, the method(s) should be part of the `Query` interface and should match Firestore's methods (aka `whereGreaterThan`, etc.).

Reply:
Yes, this was actually in my first implementation, when I was working on the search field.